src/data.jl

```
@@ -271,6 +271,25 @@ function Base.getproperty(ids::IDS, field::Symbol; to_cocos::Int=user_cocos)
     return value
 end
 
+"""
+    Base.getproperty(ids::IDSvectorGridggdTimeElement{T}, field::Symbol) where {T<:Real}
+
+This function links all grid_ggd types with each other
+
+If the grid_ggd has a path defined to another instance of grid_ggd, this instance would automatically return attributed from the referred instance.
+"""
+function Base.getproperty(ids::IDSvectorGridggdTimeElement{T}, field::Symbol) where {T<:Real}
+    if field == :path
```

Here, the condition should be:
```julia
if ismissing(ids, :path) || field == :path
```

because for the `grid_ggd` instance that actually has the data, all paths to get_property are now broken. So when `:path` is not present, it should go back to the default getproperty behavior. Additionally, if the `:path` is bad, we might want to throw an error message.